src/pat/select2/select2.js

```diff
@@ -109,8 +109,16 @@ export default Base.extend({
         await import("select2");
         try {
             // Don't load "en" which is the default where no separate language file exists.
-            if (this.options.language && this.options.language !== "en") {
-                await import(`select2/select2_locale_${this.options.language}`);
+            if (this.options.language && this.options.language !== "en" && !this.options.language.startsWith("en")) {
+                var lang = this.options.language;
+                // Fix for country specific languages
+                if (lang.split("-").length > 1) {
+                    lang =
+                        lang.split("-")[0] +
+                        "-" +
+                        lang.split("-")[1].toUpperCase();
+                }         
```

Comment:
instead of calling `.split()` 3 times I'd suggest:

```suggestion
                let lang = this.options.language.split("-");
                // Fix for country specific languages
                lang = (lang.length > 1) ? `${lang[0]}-${lang[1]}` : lang[0];
```